openstates/nc/bills.py

```diff
@@ -173,13 +199,17 @@ def scrape_bill(self, chamber, session, bill_id):
                     break
             else:
                 atype = None
-
-            bill.add_action(action, act_date, chamber=actor, classification=atype)
+            if act_date is not None:
```

Comment:
if we're skipping actions without dates, let's comment why/how often that happens

Reply:
Looking into how often this takes place. Honestly don't remember.